src/Microsoft.SqlTools.ServiceLayer/Migration/MigrationService.cs

```diff
@@ -186,203 +183,134 @@ public bool TryGetData(string column, out object value)
             }
         }
 
-        internal async Task<List<MigrationAssessmentInfo>> GetAssessmentItems(SqlObjectLocator target, string connectionString)
+        internal async Task<MigrationAssessmentResult> GetAssessmentItems(string connectionString)
         {
```

Comment:
be sure to coordinate with the following changes 
https://github.com/microsoft/sqltoolsservice/pull/1219